src/components/render-list-item.component.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-param-reassign */
+import { renderTemplate } from '../utils/render-template';
+
+export const renderListItem = (itemState) => {
+  const item = renderTemplate(`
+    <li class="list-item">
+      <label class="checkbox">
+        <input type="checkbox" class="checkbox__input" />
+        <span class="checkbox__decor">
+          <svg class="checkbox__decor-icon" fill="#fff" width="12" height="12">
+            <use xlink:href="#tick"></use>
+          </svg>
+        </span>
+      </label>
+      <div class="flexible-wrap">
+        <textarea
+          value="${itemState.text}"
+          rows="1"
+          onInput="this.parentNode.dataset.replicatedValue = this.value"
+          class="list-item__content card-form__content"
+        ></textarea>
+      </div>
+      <svg class="list-item__remove" fill="black" width="12" height="12">
+        <use xlink:href="#cross"></use>
+      </svg>
+    </li>
+  `);
+
+  const formItemText = item.querySelector('.card-form__content');
+  const formCheckboxInput = item.querySelector('.checkbox__input');
+
+  formItemText.value = itemState.text;
+
+  if (itemState.isDone) {
+    formItemText.classList.add('list-item__content--crossed');
+    formCheckboxInput.setAttribute('checked', 'true');
+    itemState.isDone = true;
```

Comment:
Why do we need assignment here? (same with L:40)

Reply:
We do not need) Removed